src/App.js

```diff
@@ -1,62 +1,66 @@
 import React, { Component } from 'react'
-import movieData from './FakeData'
+import getAPIData from './APICalls.js'
+// import movieData from './FakeData'
 import Movies from './components/Movies/Movies';
-import {Route, Switch, Redirect} from 'react-router-dom'
-import Header from './components/Header/Header'
-// import Card from './components/Card/Card';
+import Movie from './components/MovieInfo/MovieInfo';
 import './App.css';
+// import {Route, Switch, Redirect} from 'react-router-dom'
+// import { BrowserRouter } from 'react-router-dom';
+// import Header from './components/Header/Header'
+// import Card from './components/Card/Card';
+
+
+// const setEventHandler = true
 
 class App extends Component {
   constructor() {
     super()
     // this.movieData = movieData
     this.state = {
       singleMovie: null,
-      movies: []
+      movies: [],
+      isClicked: false
     }
     this.showSingleMovie = this.showSingleMovie.bind(this)
   }
 
   componentDidMount = () => {
-    this.setState({ movies: movieData.movies})
+    getAPIData('movies')
+    .then((data) => this.setState({ movies: data.movies }))
+    .catch((error) => console.log(error))
```

Comment:
Rather than console logging the error message, lets do something with it! Like setting it to state and displaying it to the user!